src/ert/config/response_config.py

```diff
@@ -1,16 +1,34 @@
 import dataclasses
 from abc import ABC, abstractmethod
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 import xarray as xr
 
-from ert.config.parameter_config import CustomDict
+from ert.config.commons import Refcase
+
+from .observation_vector import ObsVector
+from .parameter_config import CustomDict
+
+
+@dataclasses.dataclass
+class ObsArgs:
```

Comment:
Can be done in the future, but should this be a base class, and there should be a:

```python
class ObsArgs:
    obs_name: str
    values: Any
    
class SummaryArgs(ObsArgs):
    refcase: ... 
    history: ...    
```

Reply:
I think leaving it for future is a good idea, will be done in responses refactor anyway, summaryconfig is one of the most entangled parts of this